spec/services/symphony_client_spec.rb

```diff
@@ -100,11 +108,17 @@
                session_token: 'e0b5e1a3e86a399112b9eb893daeacfd')
     end
 
-    it 'returns successful + errored titles for individual renewal requests in symphony' do
-      renew_response = client.renew_items(user, checkouts)
+    it 'returns successful + errored items for individual renewal requests in symphony' do
+      renew_response = client.renew_items(user, [checkouts.first, checkouts.second])
 
-      expect(renew_response).to include error: [[checkouts.last, 'Item has holds']],
+      expect(renew_response).to include error: [[checkouts.second, 'Item has holds']],
```

Comment:
```suggestion
      expect(renew_response).to be error: [[checkouts.second, 'Item has holds']],
```

The two things being compared have equal (`eq`) values, is this more explicit/specifc?